examples/04-Icepak/Icepak_3DComponents_Example.py

```diff
@@ -175,13 +168,9 @@
 # components are not natively supported. Then it is possible to export the whole package as 3d component. Here the
 # auxiliary dictionary is needed to export monitor objects, datasets and native components.
 ipk.flatten_3d_components()
-ipk.modeler.create_3dcomponent(
-    component_file=os.path.join(temp_folder, "componentLibrary", "PCBAssembly.a3dcomp"),
-    component_name="PCBAssembly",
-    auxiliary_dict=True,
-    included_cs=["Global", "HeatsinkCS", "PCB_Assembly"],
-    reference_cs="PCB_Assembly"
-)
+ipk.modeler.create_3dcomponent(input_file=os.path.join(temp_folder, "componentLibrary", "PCBAssembly.a3dcomp"),
+                               name="PCBAssembly", coordinate_systems=["Global", "HeatsinkCS", "PCB_Assembly"],
+                               reference_cs="PCB_Assembly", export_auxiliary=True)
```

Comment:
should not be "reference_coordinate_system"

Reply:
@maxcapodi78 I kept it as is but I can change reference_cs to reference_coordinate_system